llama_stack/providers/remote/inference/fireworks/fireworks.py

```diff
@@ -264,4 +270,15 @@ async def embeddings(
         model_id: str,
         contents: List[InterleavedTextMedia],
     ) -> EmbeddingsResponse:
-        raise NotImplementedError()
+        model = await self.model_store.get_model(model_id)
+
+        client = self._get_openai_client()
```

Comment:
you can use openai client for fireworks also?! their own client doesn't have an embeddings method?

Reply:
Yes, looks like their embedding service is wire compatible with open AI client. The wired part is they do not expose this in the regular client.
https://docs.fireworks.ai/guides/querying-embeddings-models
The docs just mention open ai client to be used for embeddings.

Reply:
updated to just use regular SDK